gitversioner/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'kotlin'
+apply plugin: 'com.novoda.bintray-release'
+
+group = 'com.pascalwelsch.gitversioner'
+version = '0.0.1-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.1.2-4'
```

Comment:
code inside `buildscript` will be executed before code outside of `buildscript` which is required to get the kotlin plugin dependency. Also this is how kotlin is added everywhere, I'll stick to the standard